src/flamenco/runtime/tests/Local.mk

```diff
@@ -11,9 +11,6 @@ endif
 
 run-runtime-test: run-runtime-test-1 run-runtime-test-2 run-runtime-test-3
 
-run-runtime-native: $(OBJDIR)/unit-test/test_native_programs
-	OBJDIR=$(OBJDIR) src/flamenco/runtime/tests/run_native_tests.sh
-
```

Comment:
Since these native program tests were not showing up in the previous "coverage" reports,  are we sure they were being included?